api/src/Controller/AuthController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authentication\Provider\DaoAuthenticationProvider;
+use Symfony\Component\Security\Core\User\UserChecker;
+use Symfony\Component\Security\Core\User\InMemoryUserProvider;
+use Symfony\Component\Security\Core\Encoder\EncoderFactory;
+use Symfony\Component\Security\Core\Encoder\MessageDigestPasswordEncoder;
+use Symfony\Component\Security\Core\User\User;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+
+class AuthController extends AbstractController
+{
+
+    public function login_check(Request $request)
+    {
+        // set the in_memory provider
+        $userProvider = new InMemoryUserProvider(
+            array(
+                'admin' => array(
+                    // password is "foo"
+                    'password' => '5FZ2Z8QIkA7UTZ4BYkoC+GsReLf569mSKDsfods6LYQ8t+a8EW9oaircfMpmaLbPBh4FOBiiFyLfuZmTSUwzZg==',
```

Comment:
If you use the `PlaintextPasswordEncoder` (see below), you don't need the hash.

Reply:
ok, cool for tests ^^